components/tools/OmeroWeb/omeroweb/settings.py

```diff
@@ -1140,6 +1140,11 @@ def report_settings(module):
 # FEEDBACK_APP: 6 = OMERO.web
 FEEDBACK_APP = 6
 
+# For any given release of api, we may support
+# one or more versions of the api.
+# E.g. /api/v1.0/
+API_VERSIONS = ('0.1',)
```

Comment:
We need to have the full 'how we're going to handle versions' discussion before we change anything here. This is really just a placeholder for this PR. I'll add a TODO note in the code here.